plugin/nfs_plugin/nfs.py

```
@@ -28,14 +28,11 @@ class NFSPlugin(INfs, IStorageAreaNetwork):
     def _run_cmd(cmd):
-        try:
-            if sys.version_info[0] > 3 or \
-               (sys.version_info[0] == 3 and sys.version_info[1] >= 3):
-                subprocess.check_call(cmd, timeout=3)
-            else:
-                subprocess.check_call(cmd)
-        except:
-            raise
+        if sys.version_info[0] > 3 or \
+           (sys.version_info[0] == 3 and sys.version_info[1] >= 3):
+            subprocess.check_call(cmd, timeout=3)
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)